app/services/idv/data_url_image.rb

```diff
@@ -19,9 +19,7 @@ def read
       if base64_encoded?
         Base64.decode64(@data)
       else
-        # rubocop:disable Lint/UriEscapeUnescape
-        URI.decode(@data)
-        # rubocop:enable Lint/UriEscapeUnescape
+        ''
```

Comment:
This test expects bad data to return an empty string: https://github.com/18F/identity-idp/blob/bce24753cf74eb3938ccd6448237e14fe17c06e7/spec/services/idv/data_url_image_spec.rb#L13-L19